src/main/app/claims/models/theirs/theirDetails.ts

```diff
@@ -1,18 +1,21 @@
 import { Serializable } from 'models/serializable'
 import { Address } from 'claims/models/address'
 import { Representative } from 'claims/models/representative'
+import { HowDidYouServe } from 'forms/models/howDidYouServe'
 
 export class TheirDetails implements Serializable<TheirDetails> {
   type: string
   name: string
   address: Address
   representative: Representative
+  howWereYouServed: HowDidYouServe = new HowDidYouServe()
```

Comment:
any reason for kyepiale name different?

Reply:
it didn't make sense to me that you would type defendant.howdidyouserve so I named the variable so that when it is typed out it would be defendant.howereyouserved but the page is called how did you serve so kept the model called that.